packages/shared/src/hooks/translation/useTranslation.ts

```diff
@@ -106,6 +107,14 @@ export const useTranslation: UseTranslation = ({ queryKey, queryType }) => {
             ? !node.clickbaitTitleDetected
             : !node.sharedPost?.clickbaitTitleDetected,
         )
+        .filter(
+          (post) =>
+            !(
+              (post.type === PostType.Article ||
+                post.type === PostType.VideoYouTube) &&
+              post.language === language
```

Comment:
Just a question to confirm, we only do kvasir if language is selected right, for `null` (default) we don't?

Reply:
Yeah.
https://github.com/dailydotdev/apps/blob/cc62cd3704bdf0380e0785b0b832cad4f11eab33/packages/shared/src/hooks/translation/useTranslation.ts#L54
https://github.com/dailydotdev/apps/blob/cc62cd3704bdf0380e0785b0b832cad4f11eab33/packages/shared/src/hooks/translation/useTranslation.ts#L91-L93